### ISSUE TITLE
Bray rounds have the wrong number of ends

### ISSUE THREAD

**Issue by donbex** [DreierF/MyTargets#302]
In version 2.4.4 (70) the standard *Bray I* and *Bray II* rounds are set up as 5 ends of 6 arrows each, but they should be 10 ends of 3 arrows each.

Indeed, according to the [Archery GB Rules of Shooting](http://www.archerygb.org/wp-content/uploads/2017/05/Proc0104RoSApril2017-22490.pdf), *Bray I* and *Bray II* are indoor rounds, and as such have 3 arrow ends (see rule 403.a and table 4-1).

**Comment by jamshidrostami** [comment]
I can fix it by changing the configuration data for these standards.
do we need to update previous entered shots data?

**Comment by DreierF** [comment]
Yeah, we most probably need a migration of previously entered data 😏
